en/micro-integrator/docs/references/comparisong-mi7-ei6xx.md

```diff
@@ -218,39 +218,6 @@ The following features, which are not needed for MSA-based deployments or not us
 			Third-party offering like <b>rsync</b>
 		</td>
 	</tr>
-	<tr>
-		<td>
-			Hot deployment/Hot update
-		</td>
-		<td>
-			Services and artifacts are immutable.
-		</td>
-		<td>
-			-
-		</td>
-	</tr>
-	<tr>
-		<td>
-			Built-in Hazelcast-based clustering
-		</td>
-		<td>
-			The microservices world typically handles clustering using a container orchestration systems like Kubernetes.</br></br>However, built-in clustering support will be added to the Micro Intgrator in a future version to support standard, centralized deployment architectures.
-		</td>
-		<td>
-			-
-		</td>
-	</tr>
-	<tr>
-		<td>
-			Built-in node coordination feature
-		</td>
-		<td>
-			Coordination support for message processors, inbound endpoints, and tasks, which is built into the ESB profile, is removed from the Micro Integrator. This feature is not a requirement in the microservices world.</br></br> However, coordination support will be added to the Micro Integrator in a future version to support standard, centralized deployment architectures.
-		</td>
-		<td>
-			-
-		</td>
-	</tr>
 	<tr>
 		<td>
 			Database-based Registry
```

Comment:
RDBMS-based Registry?

Reply:
no , MI is file based as mentioned in alternative.

Reply:
Yes.. I mean.. shall we reword 'Database-based Registry' to 'RDBMS-based Registry'